torch/utils/cpp_extension.py

```diff
@@ -741,11 +746,20 @@ def load_inline(name,
 
     cpp_sources.insert(0, '#include <torch/extension.h>')
 
+    def add_safe_version(function_name):
```

Comment:
I think, for future readers, it will be helpful if there's a little comment saying what safe versions do.